index.js

```diff
@@ -141,7 +141,7 @@ function build (schema, options) {
 
   var dependencies = []
   var dependenciesName = []
-  if (hasOf(schema) || hasSchemaSomeIf) {
+  if (hasConst(schema) || hasOf(schema) || hasSchemaSomeIf) {
```

Comment:
Wouldn't it be better to check `if (!schema) { return false }` in this condition instead of doing it twice in `hasConst` and `hasOf` ?

Reply:
Good point

Reply:
In general `hasOf` and `hasConst` are same, the difference only in keys which they are looking for, so in general I think they can be united